### PR TITLE
Mu 20

### PR DESCRIPTION
**Pull request by Staroz**
Please, review this PR. Thanks.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Cypress run
         uses: cypress-io/github-action@v5
         with:
-            spec: cypress/e2e/updateCards.cy.js
+            spec: cypress/e2e/attachOnCard.cy.js
```
Comment (sphinix27) [inline .github/workflows/main.yml]: Update this to run the whole suite, you are only testing one test file, while on CICD all the tests should run

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -37,4 +37,4 @@ jobs:
       run: |
           echo '${{ secrets.CREDENTIALS_E2E }}' > cypress/fixtures/credentials1.json
     - name: Run Playwright tests
-      run: npx playwright test updateCards.spec.js
\ No newline at end of file
+      run: npx playwright test attachOnCard.spec.js
```
Comment (sphinix27) [inline .github/workflows/main.yml]: Same as above

File: .github/workflows/playwright.yml
Comment (sphinix27) [inline .github/workflows/playwright.yml:1]: Is this file required?

**Review by sphinix27 — APPROVED** [review]
LGTM, nice work. Just update those suggestions, but I'll approve.

**Review by sphinix27 — APPROVED** [review]
lgtm

**Comment by sphinix27** [comment]
Try to fix your GitLab CI

**Comment by Staroz** [comment]
Ok, I was testing Cypress Cloud, hope it's good, can you please review and authorize the merge?

File: cypress/support/apiCommands.js [hunk 3]
```diff
@@ -14,9 +14,9 @@ async function deleteWorkspaces() {
 
     for (const organization of organizations) {
 
-      await axios.delete(
-        `https://api.trello.com/1/boards/${organization.idBoards}?&key=${apiKey}&token=${apiToken}`
-      );
+      // await axios.delete(
+      //   `https://api.trello.com/1/boards/${organization.idBoards}?&key=${apiKey}&token=${apiToken}`
+      // );
```
Comment (sphinix27) [inline cypress/support/apiCommands.js:19]: If this was intended, please remove the code, not just comment it.

**Review by sphinix27 — APPROVED** [review]
LGTM, but take a look at that commented code.
